### DISCUSSION TITLE
understanding-pathbase-in-aspnetcore/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#25] [Announcements]
# understanding-pathbase-in-aspnetcore/

In this post I'll describe a lesser-known property on HttpRequest called `PathBase`. I describe what it does, when it's useful, and how to use it.

https://andrewlock.net/understanding-pathbase-in-aspnetcore/

**Comment by rcwowbagger** [comment]
Thanksyou! I've wasted so much time trying to make AspNetCore work behind nginx. This was the key to solving it!

[a comment by an automated account is not shown]

**Comment by AndrewTriesToCode** [comment]
Good information not well documented by Microsoft.
